css/index.css

```diff
@@ -63,8 +63,37 @@ ul {
 }
 
 #nav-flex{
-    display:flex;
-    flex-direction:column;
-    flex-wrap:wrap;
-    justify-content:space-between;
+    display: grid;
+    overflow: auto;
+    gap: 1px;
+    padding: 1px;
+    border: .5px solid #1f1001;
+    background-color: bisque;
+    grid-template-rows: repeat(2, 1fr);
+    grid-template-columns: repeat(3, 1fr);
+}
+
+
+
+/* Mobile Phone */
+
+@media screen and (max-width:500px){
```

Comment:
This is great! 
The assignment asks for you to make two media queries, though!

So, you can make another one for medium size screens. Your breakpoint could be something like this:
```
@media screen and (max-width:1280px) {
/* Your code here */
}
```

Here's a screenshot of the breakpoints used by a popular CSS framework called Tailwind. They use min-width instead of max-width, but take note of the sizes for different screens.

![Screenshot 2024-04-29 at 5 19 47 PM](https://github.com/laurenbgod/lauren-godwin-haumea/assets/55720928/5ab2ab50-84fa-4a59-b0d1-5f71f35f5808)